examples/grails3-hibernate5/build.gradle

```diff
@@ -5,33 +5,33 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-autoconfigure"
```

Comment:
I'm guessing the grails gradle plugin includes the grails-bom by default?  I'm not as familiar with the way the bom's work - does it just pull in the versions or does it automatically add the dependencies too?  Are any of these implementations even needed if the bom is included?

Reply:
Yes, the `grails-gradle-plugin` adds the `grails-bom` to the project.

A `bom` only provides a suggested versions for the modules it has listed, so you don't have to provide a specific version yourself. It does not add any dependencies by itself.